### PR TITLE
Refactor weld_common/ and easy_ll/ crates into modules

### PR DESCRIPTION
**Pull request by deepakn94**
This is to ensure that we can publish Weld as a single crate.

### CODE REVIEW

File: tests/integration_tests.rs [hunk 1]
```diff
@@ -2,10 +2,9 @@ use std::env;
 use std::thread;
 
 extern crate weld;
-extern crate weld_common;
 extern crate libc;
 
-use weld_common::WeldRuntimeErrno;
+use weld::weld_common::WeldRuntimeErrno;
```
Comment (mateiz) [inline tests/integration_tests.rs]: should we rename this module to `common` or just put it in the main weld `lib.rs`?

File: weld/easy_ll/tests.rs [hunk 2]
```diff
@@ -15,7 +16,7 @@ fn basic_use() {
            %2 = call i64 @bar(i64 %1)
            ret i64 %2
        }
-    ");
+    ", Some(LIB_WELD_RT));
```
Comment (mateiz) [inline weld/easy_ll/tests.rs]: You don't need to pass this in the easy_ll tests, do you? isn't it enough to pass None?

**Comment by mateiz** [comment]
Looks OK overall but I added a couple of comments

File: weld/easy_ll/tests.rs (hunk 2 above)
Comment (deepakn94) [reply weld/easy_ll/tests.rs]: Fixed.

**Comment by deepakn94** [comment]
@mateiz Addressed your comments.

File: tests/integration_tests.rs (hunk 1 above)
Comment (deepakn94) [reply tests/integration_tests.rs]: Renamed the module to `common`
